syllabus/core/curriculum_base.py

```diff
@@ -12,24 +12,27 @@ class Curriculum:
     """Base class and API for defining curricula to interface with Gym environments.
     """
 
-    def __init__(self, task_space: TaskSpace, random_start_tasks: int = 0, task_names: Callable = None) -> None:
+    def __init__(self, task_space: TaskSpace, task_names: Callable = None, use_startup_behavior: bool = False, sampled_list: List[List[Tuple[str, int, int]]] = None) -> None:
```

Comment:
what is `sampled_list` supposed to contain? I see below that its a list per environment of behaviors, episodes, and tasks. There are a few issues with that:
1. You don't need to worry about the number of environments, it's not relevant here. You simply need the `sample` function to return a list of k tasks.
2. The user is only going to choose a single behavior for the entire curriculum, they won't pick a different one per environment
3. The number of episodes is also irrelevant, since we'll keep providing tasks for as many episodes as they decide to run, and we can't know that number at initialization
4. the user won't select tasks, we have to sample them